DataFormats/TrackerRecHit2D/interface/SiPixelRecHitQuality.h

```diff
@@ -65,7 +65,7 @@ class SiPixelRecHitQuality {
     inline float probabilityXY(QualWordType qualWord) const {
       int raw = (qualWord >> probX_shift) & probX_mask;
       if (raw < 0 || raw > 16383) {
-        warningOutOfBoundRaw("XY", raw, qualWord);
+        //warningOutOfBoundRaw("XY", raw, qualWord);
```

Comment:
I thought that by default we get 0, but apparently it's not the case. Which values do we get for a not computed case?

Reply:
On the one hand, I had commented more warnings than necessary (fixed).
The only relevant warning (left commented) is in the `setProbabilityQ` function.
If not commented, warnings look like:
`ProbQ outside the bounds of the quality word: ProbQ = 1+-2 and QualityWord = 264 --> Now defaulting to ProbQ = 0.0`.
Based on the [definition of the corresponding warning function](https://github.com/trackreco/cmssw/blob/master/DataFormats/TrackerRecHit2D/src/SiPixelRecHitQuality.cc#L66), this means that `prob` equals `-1`, thus the warning is triggered.